### DISCUSSION TITLE
Change width main section

### DISCUSSION THREAD

**Discussion by maminskik** [filamentphp/filament#9199] [Help] [answered]
Hi, it is possible to change width main section in filament? I have nav section set on the top and i want have tables on full width(ofc with litte margin like header). Where i can change it?

**Comment by VincvanAs** [comment, marked as answer]
Maybe ->maxContentWidth('full'); will suite your needs in your panel provider.

Alternatively you could create a custom theme where you define the margins and styling yourself based on your own needs

https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

More information about hooks: https://filamentphp.com/docs/3.x/support/render-hooks#panel-builder-render-hooks

**Comment by blackforestcode** [comment]
@VincvanAs Thanks, that helped me, too. I've added following to my custom theme:

```css
.fi-simple-main {
    @apply sm:max-w-7xl !important;
}

```
